.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Are we going to include vscode settings on our project?

Reply:
It is not necessary. I don't know why .gitignore didn't work.